### PR TITLE
Add ausearch support to work with auditd

### PR DESCRIPTION
**Pull request by QuentinBisson**
Towards https://github.com/giantswarm/giantswarm/issues/24992

### CODE REVIEW

File: helm/fluent-logshipping-app/Chart.yaml
Comment (hervenicol) [inline helm/fluent-logshipping-app/Chart.yaml:2]: Wow, that's a big gap! No compatibility issues?

File: helm/fluent-logshipping-app/values.yaml
Comment (hervenicol) [inline helm/fluent-logshipping-app/values.yaml:29]: I guess the plugins we use are now included in the new image? And we changed the image name rather than having a custom tag?

**Review by hervenicol — APPROVED** [review]
If I understand correctly, this PR only updates the base image, the config change will come separately?

LGTM if tested.

File: helm/fluent-logshipping-app/Chart.yaml
Comment (QuentinBisson) [reply helm/fluent-logshipping-app/Chart.yaml:2]: That's what I need to test :)

File: helm/fluent-logshipping-app/values.yaml
Comment (QuentinBisson) [reply helm/fluent-logshipping-app/values.yaml:29]: The image is a custom built image from a fork https://github.com/giantswarm/fluent-bit/blob/v0.1.0/dockerfiles/Dockerfile

I had to fork the repo instead of changing the image because fluent-bit is build without the exec plugin by default so I had to enable the exec plugin and add auditd in the container to be able to have ausearch in it

**Comment by QuentinBisson** [comment]
Looks fine on installations using our custom plugins

**Comment by QuentinBisson** [comment]
Testing ongoing on viking

**Comment by QuentinBisson** [comment]
Tests are looking alright, waiting for customer feedback

**Pull request merged**
